test/pages/register-a-beacon/activity.test.tsx

```diff
@@ -1,49 +0,0 @@
-import { render, screen } from "@testing-library/react";
-import { GetServerSidePropsContext } from "next";
-import React from "react";
-import { FormJSON } from "../../../src/lib/form/formManager";
-import { handlePageRequest } from "../../../src/lib/handlePageRequest";
-import Activity, {
-  getServerSideProps,
-} from "../../../src/pages/register-a-beacon/activity";
-
-jest.mock("../../../src/lib/handlePageRequest", () => ({
-  __esModule: true,
-  handlePageRequest: jest.fn().mockImplementation(() => jest.fn()),
-}));
-
-describe("Activity", () => {
-  const activityForm: FormJSON = {
-    hasErrors: false,
-    errorSummary: [],
-    fields: {
-      activity: {
-        value: "",
-        errorMessages: [],
-      },
-      otherActivityText: {
-        value: "",
-        errorMessages: [],
-      },
-    },
-  };
-
-  it("should have a back button which directs the user to the beacon information page", () => {
```

Comment:
@zackads Please see comment below regarding checking that the page correctly renders based on the input is it provided